src/ResourceManager/Cdn/Commands.Cdn/packages.config

```diff
@@ -1,4 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-	<package id="Microsoft.Azure.Management.Cdn" version="3.0.0-preview" targetFramework="net45" />
-</packages>
+  <package id="Microsoft.Azure.Management.Cdn" version="4.0.1-preview" targetFramework="net452" />
+  <package id="Microsoft.Rest.ClientRuntime" version="2.3.10" targetFramework="net452" />
```

Comment:
@prakharsharma10 similar to the `.csproj` file, remove the references added in this file (`Microsoft.Rest.ClientRuntime.*` and `Newtonsoft.Json`) since they are referenced in a separate `packages.config`